src/lazy/binary/raw/v1_1/e_expression.rs

```diff
@@ -18,32 +18,6 @@ use crate::lazy::text::raw::v1_1::arg_group::{EExpArg, EExpArgExpr};
 use crate::lazy::text::raw::v1_1::reader::MacroIdRef;
 use crate::{try_or_some_err, v1_1, Environment, HasRange, HasSpan, IonResult, Span};
 
-#[derive(Copy, Clone)]
-pub struct BinaryEExpHeader {
```

Comment:
🗺️ This type was created to offer methods like those added in this PR, but it was never completed or used.